ibtk/include/ibtk/PETScMatUtilities.h

```diff
@@ -159,18 +130,40 @@ class PETScMatUtilities
                                               int dof_index_idx,
                                               SAMRAI::tbox::Pointer<SAMRAI::hier::PatchLevel<NDIM> > patch_level);
 
-    static inline void ib_4_interp_fcn(const double r, double* const w)
```

Comment:
Hmm? It's a standard ib_4 function, nothing special. I just fixed it. The one you had earlier worked with a shift of 1, which was confusing. We fixed that shift of 1 (i instead of i+1) in the code.